cmake/LYWrappers.cmake

```diff
@@ -87,7 +87,7 @@ define_property(TARGET PROPERTY RUNTIME_DEPENDENCIES_DEPENDS
 # \arg:AUTOGEN_RULES a set of AutoGeneration rules to be passed to the AzAutoGen expansion system
 function(ly_add_target)
 
-    set(options STATIC SHARED MODULE GEM_STATIC GEM_MODULE HEADERONLY EXECUTABLE APPLICATION IMPORTED AUTOMOC AUTOUIC AUTORCC NO_UNITY)
+    set(options STATIC SHARED MODULE GEM_STATIC GEM_SHARED GEM_MODULE HEADERONLY EXECUTABLE APPLICATION IMPORTED AUTOMOC AUTOUIC AUTORCC NO_UNITY)
```

Comment:
Can you update the function documentation above to mention the GEM_SHARED argument?

```
# \arg:SHARED (bool) defines this target to be a dynamic library while also setting the GEM_MODULE property
```

Reply:
Done (assuming you meant \arg:GEM_SHARED here)